README.md

```diff
@@ -327,17 +327,22 @@ eval $(starship init ion)
 <details>
 <summary>Nushell</summary>
 
-Add the following to the end of your Nushell configuration (find it by running `config path`):
-
-```toml
-startup = [
-  "mkdir ~/.cache/starship",
-  "starship init nu | save ~/.cache/starship/init.nu",
-  "source ~/.cache/starship/init.nu",
-]
-prompt = "starship_prompt"
+Run the following:
+
+```sh
+mkdir ~/.cache/starship
+starship init nu | save ~/.cache/starship/init.nu
 ```
 
+And add the following to the end of your Nushell configuration (find it by running `$nu.config-path`):
+
+```sh
+starship init nu | save ~/.cache/starship/init.nu
+source ~/.cache/starship/init.nu
```

Comment:
You'll need to run the `starship init nu | save ~/.cache/starship/init.nu` as a separate step. Running them back to back like this in the startup, and the init.nu won't be found (the parser is looking for "init.nu" but the evaluator won't have run the previous line.

Reply:
This is for the init file, where I believe there is no way to add additional steps. Users are instructed to run `mkdir ~/.cache/starship/` and `source ~/.cache/starship/init.nu` to partially work around this.

Reply:
For line 340 - I think this will confuse users as it looks like if they update starship and then startup nushell things will be fine, but if the newer starship has a newer init.nu, the config won't see it until it's restarted.

Maybe best to drop line 340 and just have users do the `source ~/.cache/starship/init.nu` in the config, letting them know they need to manually update after updating starship?

Reply:
It's a compromise. I think not having users update the file manually will be more reliable.